stock_inventory_line_price/models/stock_inventory.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# (c) 2016 Esther Martín - AvanzOSC
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models
+
+
+class StockInventory(models.Model):
+    _inherit = 'stock.inventory'
+
+    @api.model
+    def _get_inventory_lines(self, inventory):
+        vals = super(StockInventory, self)._get_inventory_lines(inventory)
+        for line in vals:
```

Comment:
@lasley IMO we should browse all the product at once to avoid performance issue...

Reply:
@lasley Can you look into @lmignon remarks to move forward on this ?